src/main/java/amazin/controller/ApplicationController.java

```diff
@@ -15,27 +15,27 @@
 
     @GetMapping({ "/" })
     public String index(Model model) {
-        model.addAttribute("books", bookRepository.findAll());
+        model.addAttribute("book", bookRepository.findAll());
         return "index";
     }
 
-    @GetMapping({"/login"})
-    public String login(){
+    @GetMapping({ "/login" })
+    public String login() {
         return "login";
     }
 
-    @GetMapping({"/register"})
-         public String register(){
+    @GetMapping({ "/register" })
+    public String register() {
         return "registration";
     }
 
-    @GetMapping({"/admin/register"})
-    public String registerAdmin(){
+    @GetMapping({ "/admin/register" })
+    public String registerAdmin() {
         return "registration-admin";
     }
 
-    @GetMapping({"/forgot-password"})
-    public String forgotPassword(){
+    @GetMapping({ "/forgot-password" })
+    public String forgotPassword() {
```

Comment:
Please rebase and remove the changes from this file none of them are functional changes